canlogserver.c

```diff
@@ -88,6 +88,7 @@ static volatile sig_atomic_t signal_num;
 
 void print_usage(char *prg)
 {
+	fprintf(stderr, "%s - log CAN frames and serves via TCP.\n", prg);
```

Comment:
serves?
Would "serves them" be better here?

Reply:
That's gonna be

```
$ canlogserver -h
canlogserver - log CAN frames and serves them.
   :
```
If you like it, I'm all for it.

But would it be better to read:
```
$ canlogserver -h
canlogserver - logs local CAN frames and serves to clients.
   :
```